content/en/examples/pods/config/redis-pod.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   containers:
   - name: redis
-    image: kubernetes/redis:v1
+    image: redis
```

Comment:
Maybe we should pin this to a specific tag like `redis:5.0.4`? Without a specific tag, it would be hard to diagnose any problems.

Reply:
Sure, thank you.
Will update it

Reply:
We need a tech review, as I ran through the tutorial and ran into issues. While this addresses the image issue, it does not address the larger problem of the tutorial. 

See: 

#14132